views/newsletter/editor.html

```diff
@@ -409,6 +409,10 @@ <h1 style="display:none"><%= __('Newsletter Editor') %></h1>
     'template': __('Template'),
     'designer': __('Designer'),
     'send': __('Send'),
+    'canUndo': __('Undo'),
+    'canNotUndo': __('No past history available.'),
+    'canRedo': __('Redo'),
+    'canNotRedo': __('No future history available.'),
```

Comment:
The two longer phrases may need some context that they belong to "undo/redo" feature. Also it should have domain parameter with `mailpoet` (https://developer.wordpress.org/reference/functions/__/) although this file seems to have none.